userguide/Makefile

```
@@ -133,7 +133,7 @@ latexpdf pdf:	requirepdf preproc
 	#|help-pin|
 	@sed -i '' -e 's%%{\\material\\symbol{"F703}}%g' $(PREPROCDIR)/$(BUILDDIR)/latex/*.tex
 	#|help-unpin|
```

This should be renamed to |help-close|.
```suggestion
	#|help-close|
```